TabulateSmarterTestContentPackage/Validators/FileValidator.cs

```diff
@@ -25,5 +62,21 @@ public static void Validate(ItemContext it)
             }
         }
 
+        static readonly byte[] s_expectedPrnHeader = { 0x1B, 0x04 };
```

Comment:
Add a comment of where the source or reference of what a valid PRN header value is can be found.